tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -56,7 +60,8 @@ internal TracerManager CreateTracerManager(ImmutableTracerSettings settings, Tra
                 logSubmissionManager: previous?.DirectLogSubmission,
                 telemetry: null,
                 discoveryService: null,
-                dataStreamsManager: null);
+                dataStreamsManager: null,
+                remoteConfigurationManager: previous?.RemoteConfigurationManager);
```

Comment:
If we're reusing the static RCM here, that means we won't be responding to changes in exporter settings etc, which was part of the reason for moving to `TracerManagerFactory`? 🤔

Reply:
🤦  right, forgot to change this part